ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,64 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
+    description: Compatibility package with bitnami/nginx-ingress-controller image
+    dependencies:
+      provides:
+        - ingress-nginx-controller-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash # https://github.com/bitnami/charts/blob/db53e622ecf1be5a78d1ce683cec8baa41110fa3/bitnami/nginx-ingress-controller/templates/controller-deployment.yaml#L89
+        - gmp
+        - gnutls
+        - ingress-nginx-controller-compat-${{vars.nginx-ingress-major-minor}}
+        - libffi
+        #- libhogweed6 missing pkg/so
+        - libmaxminddb-dev
+        - librtmp
+        - libssh
+        - libtasn1
+        - nettle
+        - p11-kit
+        - procps
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: nginx-ingress-controller
+          version-path: ${{vars.nginx-ingress-major}}/debian-12
+      - name: Ensure symlinks
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/common/bin
+          ln -s /usr/bin/mmdblookup ${{targets.subpkgdir}}/opt/bitnami/common/bin/mmdblookup
+          ln -s /usr/lib ${{targets.subpkgdir}}/opt/bitnami/common/lib
+          ln -s /usr/include ${{targets.subpkgdir}}/opt/bitnami/common/include
```

Comment:
we should cover this as well else the image will run into errors without this. The redirection needs to be done at package level. 
```yaml
ln -sf /dev/stdout /var/log/nginx/access.log
ln -sf /dev/stderr /var/log/nginx/error.log
```

Reply:
Thanks @kranurag7, that's covered by the main compat package which is a dependency